.gitignore

```diff
@@ -10,3 +10,4 @@ build
 dist
 *.pyc
 .pytest_cache/
+dump.rdb
```

Comment:
Do we know what this was about?

Reply:
This is the file created by running `redis-server` in the same folder as the project.